client/src/components/navbar.js

```diff
@@ -10,7 +10,7 @@ class NavBar extends Component {
         <Navbar.Brand href="/">Memestagram</Navbar.Brand>
         <Nav className="mr-auto">
           <Nav.Link href="/">Home</Nav.Link>
-          <Nav.Link href="#willeventuallybetheprofile">My Profile</Nav.Link>
+          <Nav.Link href="/profile">My Profile</Nav.Link>
```

Comment:
Do the links work on this part? On the previous update, I was not being redirected, just curious to know if that was fixed